dts/arm/st/l0/stm32l053.dtsi

```diff
@@ -32,42 +19,6 @@
 			label = "SPI_2";
 		};
 
-		timers6: timers@40001000 {
-			compatible = "st,stm32-timers";
-			reg = <0x40001000 0x400>;
-			clocks = <&rcc STM32_CLOCK_BUS_APB1 0x00000010>;
-			interrupts = <17 0>;
-			interrupt-names = "global";
-			status = "disabled";
-			label = "TIMERS_6";
-
-			pwm {
-				compatible = "st,stm32-pwm";
-				status = "disabled";
-				st,prescaler = <0>;
-				label = "PWM_6";
-				#pwm-cells = <3>;
-			};
-		};
-
-		timers22: timers@40011400 {
-			compatible = "st,stm32-timers";
-			reg = <0x40011400 0x400>;
-			clocks = <&rcc STM32_CLOCK_BUS_APB2 0x00000020>;
-			interrupts = <22 0>;
-			interrupt-names = "global";
-			status = "disabled";
-			label = "TIMERS_22";
-
-			pwm {
-				compatible = "st,stm32-pwm";
-				status = "disabled";
-				st,prescaler = <0>;
-				label = "PWM_22";
-				#pwm-cells = <3>;
-			};
-		};
-
 		usart1: serial@40013800 {
```

Comment:
usart1 is already described in stm32l051

Reply:
Fixed